src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -524,3 +534,58 @@ def __publish_new_namespaces(self, namespaces: List[NamespaceInfo]):
 
     def get_global_config(self) -> dict:
         return self.registry.get_global_config()
+
+    def __update_node(self, new_node: Node, operation: K8sOperationType):
+        # cant get the extra pods discovery info.
+        if operation == K8sOperationType.CREATE:
+            return
+
+        with self.services_publish_lock:
+            name = new_node.metadata.name
+            cache = self.__nodes_cache.get(name, None)
+            if cache is None:
+                return
+
+            if cache.resource_version >= int(new_node.metadata.resourceVersion or 0):
+                return
+
+            new_info = self.__from_api_server_node(new_node, [])
+            new_info.memory_allocated = cache.memory_allocated
```

Comment:
can the memory allocation/cpu/pods/count change on updates?
Like if a pod on a node is deleted wouldn't that change all the above?